FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -378,5 +354,18 @@ typedef void (^FIRRemoteConfigUpdateCompletion)(FIRRemoteConfigUpdate *_Nullable
                    userDefaults:(nullable NSUserDefaults *)userDefaults
                       analytics:(nullable id<FIRAnalyticsInterop>)analytics;
 
+/// Firebase INTERNAL use only!
+/// Returns the FIRRemoteConfig instance for your namespace and for the default 3P developer's app.
+/// This singleton object contains the complete set of Remote Config parameter values available to
+/// the app, including the Active Config and Default Config. This object also caches values fetched
+/// from the Remote Config Server until they are copied to the Active Config by calling
+/// activateFetched. When you fetch values from the Remote Config Server using the default Firebase
+/// namespace service, you should use this class method to create a shared instance of the
+/// FIRRemoteConfig object to ensure that your app will function properly with the Remote Config
+/// Server and the Firebase service.
++ (FIRRemoteConfig *)remoteConfigWithFIRNamespace:(NSString *)remoteConfigNamespace
+                                              app:(FIRApp *)app
+    NS_SWIFT_NAME(remoteConfig(FIRNamespace:app:));
+
```

Comment:
What may work is keeping this private but marking it's Swift port with `@objc`. Then adding an ObjC class extension in the performance file that needs it to provide the interface. The runtime should find its implementation.

Reply:
Thanks! Seems to work. Also did this for the Value_internal.